pkgs/build-support/dotnet/build-dotnet-module/default.nix

```diff
@@ -136,6 +136,9 @@ in stdenvNoCC.mkDerivation (args // {
 
       mkdir -p "$HOME/nuget_pkgs"
 
+      ${dotnet-sdk}/bin/dotnet tool restore
```

Comment:
Is there a way to specify the location to restore to? If not, we should probably symlink instead of copying below this.